pkgs/development/ocaml-modules/kicadsch/default.nix

```
@@ -0,0 +1,23 @@
+{ lib
+, buildDunePackage
+, fetchurl
+}:
+
+buildDunePackage rec {
+  pname = "kicadsch";
+  version = "0.9.0";
+
+  minimalOCamlVersion = "4.07";
+
+  src = fetchurl {
+    url = "https://github.com/jnavila/plotkicadsch/releases/download/v${version}/plotkicadsch-${version}.tbz";
+    sha256 = "sha256-B+vnEPyd3SUzviTdNoyvYk0p7Hrg/XTJm8KxsY8A4jQ=";
+  };
+
+  meta = with lib; {
+    homepage = "https://github.com/jnavila/plotkicadsch";
+    description = "Library for exporting Kicad Sch files to SVG pictures";
+    license = licenses.gpl2Only;
```

```suggestion
    license = licenses.isc;
```

https://github.com/jnavila/plotkicadsch/blob/master/kicadsch.opam#L12